modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -41,15 +41,20 @@ The format defined here is recommended because it is the most stable and versati
 For more information about Asciidoc anchors, see the link:http://asciidoctor.org/docs/user-manual/#anchordef[Asciidoctor User Manual].
 --
 
-File names:: Give the module file the same name as the anchor used in it (which is the same as or similar to the module heading), also separated by dashes. Add a prefix with an underscore to the file name to indicate the module type in the format `prefix_file-name`. Use `con_` for concept, `ref_` for reference, `proc_` for procedure, and `assembly_` for assembly.
+File names:: Give the module file the same name as the anchor used in it (which is the same as or similar to the module heading). Assembly and module file names should accurately and closely reflect the title of the assembly or module.
++
+[NOTE]
+====
+These guidelines for file names are _recommended_. Project teams may adapt or deviate from these guidelines if need be.
```

Comment:
@vikram-redhat, I think your note's a bit confusing. It makes it sound like making the file name include the anchor is recommended, and I'm not sure that's your intent.

 I'd move the note under the examples and make the text something like:

"While the module file name must include the anchor name, you can annotate the file names to differentiate the modules types. If you add annotations, ensure that all members of your team use a consistent naming style.

For example, you can a prefix with an underscore to the file name to indicate the module type in the format \`prefix_file-name\`. Use \`con_\` for concept, \`ref_\` for reference, \`proc_\` for procedure, and \`assembly_\` for assembly."

Then, you can repeat the examples and include the prefixes, if you want.

Reply:
@kalexand-rh, I agree. I addressed that in my proposed change in my comment in this PR. Does that suite what you're going for as well?

Reply:
@kalexand-rh, would you mind proposing your version of that section, similar to what I've done but how you think it should be? Always easier to have concrete proposals and then decide accordingly.

Reply:
What if we changed the word _recommended_ to _suggested?_ That should clear the ambiguity IMO.

Reply:
@tradej, s/guidelines/examples would be closer, I think. 

@sterobin, I was happy with my suggestion to the PR, but I might rewrite it like this:

```
File names:: Give the module file the same name as the anchor used in it, which is the same as or similar to the module heading. Assembly and module file names must accurately and closely reflect the title of the assembly or module.
+
While the module file name must include the anchor name, you can annotate the file names to differentiate the modules types. For example, you can a prefix with an underscore to the file name to indicate the module type in the format `prefix_file-name`. Use `con_` for concept, `ref_` for reference, `proc_` for procedure, and `assembly_` for assembly.
+
[NOTE]
====
Ensure that all members of your team use the same file naming conventions.
====
 +
 .Examples of basic file names
* `guided-decision-tables.adoc`  (Concept module)
* `creating-guided-decision-tables.adoc`  (Procedure module for creating)
* `editing-guided-decision-tables.adoc`  (Procedure module for editing)
* `guided-decision-table-examples.adoc`  (Reference module with examples)
* `guided-decision-table-columns.adoc`  (Reference module with column types)
* `guided-decision-tables.adoc`  (Assembly of guided decision table modules)
+
.Examples of annotated file names
* `con_guided-decision-tables.adoc`  (Concept module)
* `proc_creating-guided-decision-tables.adoc`  (Procedure module for creating)
* `proc_editing-guided-decision-tables.adoc`  (Procedure module for editing)
* `ref_guided-decision-table-examples.adoc`  (Reference module with examples)
* `ref_guided-decision-table-columns.adoc`  (Reference module with column types)
* `assembly_guided-decision-tables.adoc`  (Assembly of guided decision table modules)
```

Reply:
@kalexand-rh, thanks for the concrete proposal. Your "examples of basic file names" still illustrate the issue that module type annotations are there to resolve, which is that the concept file and the assembly file are identical without the annotation. Even in the basic file name scenario, annotating or otherwise distinguishing at least the assembly file is wise and often (like here) required. So we're back to my initial comment above.

Reply:
+1 to keeping the example basic as recommended. If teams want to add prefixes, they can make the choice to do so, but for the sake of standardization, it's best to be clear what the recommendation is and leaving it at that.

Reply:
I agree. I don't think we need to offer examples about prefixes. If we do that, we will have another camp about why not offer suggestions for suffixes and then we will be back at PR #25. :)

Let's leave it at the basic examples. The idea here is to standardize that the file name MUST  accurately and closely reflect the headings and titles. Over and above that, teams can do what they want with the name.

Reply:
@vikram-redhat your changes look good :). @tradej, @rkratky, is it correct that assembly files and modules will need to be in the same folder based on information from our tooling team? if so then a prefix on the assemblies makes sense.
I ask this because the [fedora quick docs repository structure](https://pagure.io/fedora-docs/quick-docs/blob/master/f/en-US) would make an excellent alternative to using prefixes.

Reply:
@tylerkelly13 AFAIK, there's no requirement from the tooling team regarding the placement of modules and assemblies.

I also like a structure where modules are in a subdirectory of the directory where assemblies are. But I also think this should be left to indiv. teams to decide.

Reply:
Is there a reason the historical decision was made to keep the assemblies and modules in the same folder? Modules in a separate folder and assembles in the parent folder make a lot of sense in terms of ease of finding the right sort of file (assembly vs content).

Reply:
@mishaone I don't think any decision of that sort was made -- the guide doesn't talk about directory structure. But the mod doc example suggests a subdirectory for modules: https://redhat-documentation.github.io/modular-docs/#a-simple-example-of-a-modular-guide